caput/misc.py

```diff
@@ -9,20 +9,16 @@
 
     vectorize
 """
-# === Start Python 2/3 compatibility
-from __future__ import absolute_import, division, print_function, unicode_literals
-from future.builtins import *  # noqa  pylint: disable=W0401, W0614
-from future.builtins.disabled import *  # noqa  pylint: disable=W0401, W0614
-from future.utils import PY3
-from past.builtins import basestring
-
-# === End Python 2/3 compatibility
 
 import collections
 import importlib
-import inspect
 import os
 
+# TODO: remove this. This was to support a patching of this routine to support Python 2
+# that used to exist in here. This will be removed when all other repos are changed to
+# use the version from `inspect`
+from inspect import getfullargspec
```

Comment:
Let's explicitly warn about this being deprecated when it's used:
for example
```
from deprecated import deprecated

@property
@deprecated(version='21.1.0', reason="""
    Use inspect.getfullargspec instead.
    This was to support a patching of this routine to support Python 2 that 
    used to exist in here and will be removed in version 21.6.0
""")
def getfullargspec()
    return inspect.getfullargspec
```

Or you could also use https://docs.python.org/3/library/exceptions.html#DeprecationWarning

Reply:
Done